style.css

```diff
@@ -52,7 +52,36 @@ p {
 }
 
 .menu-links.active {
+  position: absolute;
+  left: 0;
+  padding-top: 6vw;
   display: flex;
+  font-size: 3.2rem;
+  font-weight: 600;
+  background-color: white;
+  min-height: 85vh;
```

Comment:
- Currently, your design is not taking the whole height of the device. please, let's fix this to improve our design as much as we can as see in the Figma design. see the images below:

*Figma design*
![Screenshot from 2023-03-06 23-15-38](https://user-images.githubusercontent.com/59397159/223252547-331a333f-0d17-4c32-ac3f-2f6d41eef1d0.png)

*Your design*
![Screenshot from 2023-03-06 23-16-37](https://user-images.githubusercontent.com/59397159/223252524-1dea08a5-e16d-4e23-b17f-8feb55ac856a.png)